scrapers/AssumeThePositionStudios.yml

```diff
@@ -35,9 +47,9 @@ jsonScrapers:
           selector: data.scene.description
           postProcess:
             - replace:
-                - regex: (.*?)([A-Z,'\s]+)\.?$
+                - regex: (.*?)([A-Z,/'\s]+)\.?$
```

Comment:
I think this fix also needs to be applied on line 35, the regex for description: if I'm reading it right that's supposed to filter out the trailing list of performers and tags that are all in uppercase, but some tags contain forward slashes so it fails for [this scene](https://worstbehaviorproductions.com/trailer?updateId=12016) for example